entries/b/bookbeat.fi.json

```
@@ -0,0 +1,17 @@
+{
+  "BookBeat": {
+    "domain": "bookbeat.fi",
+    "url": "https://www.bookbeat.fi/",
+    "contact": {
+      "email": "tuki@bookbeat.com",
+      "facebook": "bookbeatfi",
+      "language": "fi"
+    },
+    "keywords": [
+      "entertainment"
+    ],
+    "regions": [
+      "fi"
```

```suggestion
      "fi",
      "at",
      "be",
      "bg",
      "hr",
      "cy",
      "cz",
      "dk",
      "ee",
      "fr",
      "de",
      "gr",
      "hu",
      "ie",
      "it",
      "lv",
      "lt",
      "lu",
      "mt",
      "nl",
      "no",
      "pl",
      "pt",
      "ro",
      "sk",
      "si",
      "es",
      "se",
      "ch",
      "gb"
```

BookBeat has regional sites as well